.gitignore

```diff
@@ -12,4 +12,11 @@ website/node_modules
 website/i18n/*
 !website/i18n/en.json
 
+v2-website/translated_docs
+v2-website/build/
+v2-website/yarn.lock
+v2-website/node_modules
+v2-website/i18n/*
+!v2-website/i18n/en.json
```

Comment:
Please refactor these to use the new folder structure.

Reply:
fixed in 4f8d81d971a4ee1142b87a86d8b1fda3c4fa8025